README.md

```diff
@@ -0,0 +1,5 @@
+To run the docker container: 
+  docker run -it -p 3000:8080 symi-backend
+
+To drop migrations from aws:
+  docker run -v /${PWD}/app/migration:/migrations --network host migrate/migrate -path=/migrations/ -database "mysql://symidb:symi-database@tcp(symi-db.chzk1jthuvd3.ap-northeast-1.rds.amazonaws.com:3306)/symi_test" drop
```

Comment:
We don't need to add database info here. Everyone can see it. It's dangerous.

Reply:
Oh yeah, those were jsut notes for me..
Will remove if from the readme